clappr/src/main/kotlin/io/clappr/player/plugin/Loader.kt

```diff
@@ -29,13 +29,24 @@ class Loader(extraPlugins: List<KClass<out Plugin>> = emptyList(), extraPlayback
             val pluginName = (pluginClass.companionObjectInstance as? NamedType)?.name
             pluginName?.let {
                 if (pluginName.isNotEmpty()) {
-                    registeredPlugins.put(pluginName, pluginClass)
+                    registeredPlugins[pluginName] = pluginClass
                     return true
                 }
             }
             return false
         }
 
+        @JvmStatic
+        fun unregisterPlugin(pluginClass: KClass<out Plugin>): Boolean {
```

Comment:
Hi guys. What do you think about let this code a little more idiomatic?
```suggestion
        fun unregisterPlugin(pluginClass: KClass<out Plugin>) =
                (pluginClass.companionObjectInstance as? NamedType)?.run {
                    name?.takeIf { it.isNotEmpty() && registeredPlugins.containsKey(it) }?.let {
                        registeredPlugins.remove(it) != null
                    }
                } ?: false
```